history-of-quakers/modernized/15-chapter-13.adoc

```diff
@@ -1031,7 +1119,8 @@ before the judges and justices aforesaid: the indictment being read, the judge s
 Mr. Crook, You have heard your indictment, what say you?
```

Comment:
Even though it doesn't say "Judge:" let's mark this paragraph as a [.dp] as well. There just won't be any italics on the line below, but that's fine. 👍